Lucidtech/Las/Client.cs

```diff
@@ -439,6 +445,10 @@ public object UpdateDocument(string documentId, List<Dictionary<string, string>>
                 queryParams.Add("consentId", consentId);
             }
 
+            if (datasetId != null) {
+                queryParams.Add("datasetId", datasetId);
```

Comment:
IsNullOrEmpty?

Reply:
It could be, currently we're a bit inconsistent here.